### DISCUSSION TITLE
Livewire wireUI Modal Doesn't Work

### DISCUSSION THREAD

**Discussion by bakarbsa** [livewire/livewire#5552] [1. Help]
Hello I'm using Laravel 9, TailwindCSS, Alphine 3, Livewire, and WireUI, When i'm using modal component from WireUI there is an error "Undefined variable $_instance". How to fix it?

![image](https://user-images.githubusercontent.com/81614254/218293315-ab70c3e8-e32c-4839-b2ac-616b12909ce3.png)

**Comment by joshhanley** [comment]
@bakarbsa can I suggest raising this issue on the WireUI repo first, as it looks like it may be an issue with the package or how you are using the package? Then if it is an underlying Livewire issue, then come back here and update this discussion. Thanks!

**Reply by bakarbsa** [reply]
Okay, thanks for the suggestion
